meta/classes/insane.bbclass

```diff
@@ -1347,7 +1347,7 @@ python () {
     for i in issues:
         package_qa_handle_error("pkgvarcheck", "%s: Variable %s is set as not being package specific, please fix this." % (d.getVar("FILE"), i), d)
 
-    if 'native-last' not in (d.getVar('INSANE_SKIP') or "").split():
+    if 'native-last' not in (d.getVar(d.expand('INSANE_SKIP_${PN}')) or "").split():
```

Comment:
Other instances of this check use this format:
`(d.getVar('INSANE_SKIP_' + package) or "")`
Stick to that for consistency?

Reply:
I tried that initially. `package` isn't defined within the scope of this anonymous python function - which runs as soon as you inherit `insane.bbclass`. I would have to decode the package name from `$PN` anyway, I think. And elsewhere in this same function, the `d.expand()` approach is used.